jsk_pcl_ros_utils/CMakeLists.txt

```diff
@@ -298,6 +299,7 @@ if (CATKIN_ENABLE_TESTING)
   add_rostest(test/point_indices_to_cluster_point_indices.test)
   if("$ENV{ROS_DISTRO}" STRGREATER "hydro")
     # not build on hydro
+    add_rostest(test/add_point_indices.test install_test_data)
```

Comment:
according to doc http://docs.ros.org/jade/api/catkin/html/howto/format1/rostest_configuration.html, you need DEPENDENCIES
```
add_rostest(test/add_point_indices.test DEPENDENCIES install_test_data)
```